es/criteria/open-standards.md

```diff
@@ -23,23 +23,23 @@ order: 11
 
 * Hacer entendible cómo se puede utilizar el software.
 
-## Cómo testear
+## Cómo probar
```

Comment:
I would like to keep the word "Testear". It is in the RAE dictionary (https://dle.rae.es/testear?m=form) and also refers directly to the word 'test', as in perform software checks. Perhaps we can use "hacer pruebas o tests"